### PR TITLE
Issues/558

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
Add network state monitoring.

### CODE REVIEW

File: Trust/UI/NetworkStateView.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                return NSLocalizedString("network.state.good", value: "Good", comment: "Good")
+            case .bad:
+                return NSLocalizedString("network.state.bad", value: "Bad", comment: "Bad")
+            }
+        }
+
+        func color() -> UIColor {
+            switch self {
+            case .good:
+                return UIColor.green
```
Comment (vikmeup) [inline Trust/UI/NetworkStateView.swift]: Use `Colors` struct

File: Trust/Transactions/ViewControllers/TransactionsViewController.swift [hunk 2]
```diff
@@ -98,6 +104,12 @@ class TransactionsViewController: UIViewController {
 
         navigationItem.titleView = titleView
         titleView.viewModel = BalanceViewModel()
+
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        chaineStateObservation()
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(customView: networkStateView)
```
Comment (vikmeup) [inline Trust/Transactions/ViewControllers/TransactionsViewController.swift]: I think we wanted to have this view on Settings view for now?

Comment (OlegGordiichuk) [reply Trust/Transactions/ViewControllers/TransactionsViewController.swift]: Done

File: Trust/UI/NetworkStateView.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/UI/NetworkStateView.swift]: Done

**Pull request merged**